agent/api/src/main/java/org/apache/shardingsphere/agent/api/plugin/AgentPluginEnable.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.agent.api.plugin;
+
+public interface AgentPluginEnable {
+    
+    /**
+     * Is the plugin enabled.
+     *
+     * @param isEnhancedForProxy is enhanced for proxy
+     * @return true or false
+     */
+    default boolean isPluginEnabled(boolean isEnhancedForProxy) {
```

Comment:
Is it only for Proxy?  How to check plugin enabled in JDBC?

Reply:
True represents the `Proxy` environment, False represents the `JDBC` environment